.dependabot/config.yml

```diff
@@ -54,6 +54,8 @@ update_configs:
           dependency_name: "io.quarkus.http:quarkus-http-servlet"
       - match:
           dependency_name: "io.quarkus.http:quarkus-http-websockets-jsr"
+      - match:
+          dependency_name: "io.quarkus:quarkus-registry-descriptor"
```

Comment:
remind me again why the registry descriptor api is external ?

Reply:
i.e. quarkus-registry-descriptor depends on quarkus... (see https://github.com/quarkusio/quarkus-registry-descriptor/blob/master/pom.xml#L61) but now quarkus itself depends on quarkus-registry-descriptor... which one do we build first ?

Reply:
> remind me again why the registry descriptor api is external ?

This was the result from the conversation we had with @aloubyansky some time ago. Unless I misunderstood, he suggested putting in a separate repository and releasing it separately


> now quarkus itself depends on quarkus-registry-descriptor... which one do we build first ?

`quarkus-registry-descriptor` has a hard dependency on `quarkus-platform-descriptor-api` which is a shared dependency in `quarkus-devtools-common` (which depends on `quarkus-registry-descriptor`). So the build order is: 

- `quarkus-platform-descriptor-api` 
- `quarkus-registry-descriptor`
- `quarkus-devtools-common`


Perhaps it would make sense to move this code under https://github.com/quarkusio/quarkus/tree/master/independent-projects/tools?

Reply:
The projects referenced in that pom are actually under `independent-projects`, btw.

Reply:
yes, thus why not put it there ? with the current setup you need to first build indepdnent projects then build registry then build the rest?